ms_client/conf.py

```diff
@@ -46,4 +46,9 @@
 
     # Maximum number of files per request
     'UPLOAD_MAX_FILES': 100,
+
+    # External SMTP server config
+    'SMTP_SERVER': None,
+    'SMTP_EMAIL': None,
```

Comment:
We must split SMTP_EMAIL from SMTP_LOGIN (i.e. yes gmail uses the email as login, but not every service), i only found out during testing sorry